packages/visual-service/tests/utils.test.ts

```diff
@@ -334,31 +356,33 @@ describe('utils', () => {
                 },
                 isAndroid: true,
                 isMobile: true,
-                getWindowSize: vi.fn().mockResolvedValueOnce({ width: 100, height: 200 }),
+                execute: vi.fn().mockResolvedValueOnce({ realDisplaySize:'100x200' }),
             })
-            expect(await getInstanceData(driver)).toMatchSnapshot()
+            expect(await getInstanceData({ currentBrowser: driver, initialDeviceRectangles: DEVICE_RECTANGLES, isNativeContext:true })).toMatchSnapshot()
         })
 
         it('should return instance data when the lambdatest capabilities are provided', async() => {
             const driver = createDriverMock({
-                ...DEFAULT_DESKTOP_BROWSER,
+                capabilities: {
+                    ...DEFAULT_DESKTOP_BROWSER.capabilities,
+                    // @ts-ignore
```

Comment:
can't we fix this instead?

Reply:
Yes, I can take a look at it for WDIO, but not part of this PR. It's already a big ass one :D